pyproject.toml

```
@@ -18,6 +18,7 @@ maintainers = [
 ]
 
 dependencies = [ # Migration from requirements.txt
+    "torch<2.5.0",
```

Can remove this if this is merged after the remove of ocf_datapipes

---

removed

---

```suggestion
    "torch",
```
still shows up on changed files, so just to double-check?

---

fixed now